runtime/convertValues.go

```diff
@@ -206,14 +216,15 @@ func exportCompositeValue(
 	fieldNames := t.CompositeFields()
 	fields := make([]cadence.Value, len(fieldNames))
 
-	fieldsMap := v.Fields()
 	for i, field := range fieldNames {
 		fieldName := field.Identifier
-		fieldValue, ok := fieldsMap.Get(fieldName)
 
-		if !ok && v.ComputedFields != nil {
-			if computedField, ok := v.ComputedFields.Get(fieldName); ok {
-				fieldValue = computedField(inter)
+		// TODO: provide proper location range
+		fieldValue := v.GetField(fieldName)
+		if fieldValue == nil && v.ComputedFields != nil {
+			if computedField, ok := v.ComputedFields[fieldName]; ok {
+				// TODO: provide proper location range
+				fieldValue = computedField(inter, interpreter.ReturnEmptyLocationRange)
```

Comment:
Follow-up task: It would be great to pass the location range (as a getter) to the export, so that when the export fails, the error message can provide proper location information to the user. (It would point to the return statement of the script)